doc/devel/documenting_mpl.rst

```diff
@@ -888,6 +888,42 @@ should ideally be named similar to :file:`imshow_mynewexample.py`.
 Miscellaneous
 =============
 
+Moving documentation
+--------------------
+
+Sometimes it is desirable to move or consolidate documentation.  With no
+action this will lead to links either going dead (404) or pointing to old
+versions of the documentation.  Preferable is to make a redirect on the
+new page that refers to the old page.  This will make the old page, but it
+will have an html refresh that immediately redirects the viewer to the new
+page.
```

Comment:
```suggestion
versions of the documentation.  It is preferable is to make a redirect on the
old page that refers to the new page, which will have an html refresh that
immediately redirects the viewer to the new page.
```

Reply:
The redirect is on the page that still exists, the "new page" - the old rat is removed.

Reply:
The redirect _directive_ is on the new page, but the redirect itself is on the (generated) old page. You can swap which one you want to refer to in this sentence from what I suggested, but at the moment, it is incorrect.

Reply:
Changed this paragraph to:

> Sometimes it is desirable to move or consolidate documentation.  With no
action this will lead to links either going dead (404) or pointing to old
versions of the documentation.  Preferable is to replace the old page
with an html refresh that immediately redirects the viewer to the new
page. So, for example we move ``/doc/topic/old_info.rst`` to
``/doc/topic/new_info.rst``.  We remove ``/doc/topic/old_info.rst`` and
in ``/doc/topic/new_info.rst`` we insert a ``redirect-from`` directive that
tells sphinx to still make the old file with the html refresh/redirect in it
(probably near the top of the file to make it noticeable)